### PR TITLE
added search ftps to code section

### LINKED ISSUE #163: Add https://www.searchftps.net/ to code section
Add https://www.searchftps.net/ to code section

### PR DESCRIPTION
**Pull request by Virtual4087**
Closes #163

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -161,6 +161,7 @@ A curated list of awesome search engines useful during Penetration testing, Vuln
 - [Snipplr](https://snipplr.com/all) - Code snippet search engine that allows users to search and share code snippets across various programming languages and frameworks
 - [Postman Public Collections](https://www.postman.com/explore/collections) - Explore the best APIs, collections, workspaces in the world on the Postman Public API Network
 - [ScriptMafia](https://scriptmafia.org) - Download full nulled scripts
+- [SearchFTPs](https://www.searchftps.net/) - Download files located on public FTP servers
```
Comment (edoardottt) [inline README.md:164]: The description should be `The most advanced FTP Search Engine service maintained by members`

**Review by edoardottt — CHANGES_REQUESTED** [review]

**Review by Virtual4087 — COMMENTED** [review]
ok i'll change it

File: README.md (hunk 1 above)
Comment (Virtual4087) [reply README.md]: done

**Review by edoardottt — APPROVED** [review]
lgtm

**Pull request merged**